### PR TITLE
Add GPS diagnostics

### PR DESCRIPTION
**Pull request by lask**

### CODE REVIEW

File: src/diagnostics.toit [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be found
+// in the LICENSE file.
+
+/**
+Diagnostics information for a GNSS receiver.
+*/
+class GnssDiagnostics:
```
Comment (andersjohnsen) [inline src/diagnostics.toit:8]: Perhaps just diagnostics? It's already scoped to the package (`ublox_gnss.Diagnostics`).

File: src/diagnostics.toit [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  /**
+  The time to first fix.
+  0, if no fix has been obtained.
+  */
+  time_to_first_fix/Duration
+
+  // TODO(Lau): Detail this measure.
```
Comment (andersjohnsen) [inline src/diagnostics.toit:15]: Perhaps we should just omit this part, until we learn how GNSS signal quality is really computed?

**Review by andersjohnsen — APPROVED** [review]
LGMT!

File: src/diagnostics.toit (hunk 2 above)
Comment (lask) [reply src/diagnostics.toit:15]: We compute the signal quality (and it is the average of the cno of the first 4 satellites).

**Pull request merged**